### DISCUSSION TITLE
[Feature Request]: How to add/edit meetings that will remain electronic?

### DISCUSSION THREAD

**Discussion by OSAwebsite** [code4recovery/12-step-meeting-list#1251] [Q&A] [answered]
### Contact Details

os.website.a86@gmail.com

### Requested Feature/Enhancement

Can you let me know how you want electronic meetings to be registered?
Especially when it comes to the 'location' details.
Al-Anon has voted in several options for electronic groups.
Only two will apply to Ontario South.
Meetings can be part of a new electronic district, or they can stay a part of the physical district they originally came grom.
In both cases there will be no address.

Anything else besides the address that is specific to these types of meetings?

**Comment by joshreisner** [comment, marked as answer]
hi @OSAwebsite we ask that plugin users create discussions, rather than issues, thanks. i've converted this issue to a discussion

to list a meeting in our plugin it needs to have some kind of geographic place associated. in the case of online-only meetings, we recommend an approximate location within your service area, like `Ontario, Canada` or `Toronto, ON, Canada`

**Reply by WealthySoul** [reply]
Oops, I think I have the same problem and I created and issue. Sorry I didn't know. 
But the approximate location errored out on me. It won't save the draft when putting in the approximate location.

**Reply by joshreisner** [reply]
hi @WealthySoul yes, in your case i think you have "Can I attend this meeting in-person" set to Yes which is why it doesn't like the approximate location. If you set  that to No it should resolve the error

**Comment by OSAwebsite** [comment]
Okay, good to know. Will try that when it becomes relevant.
